phoenix-core/src/test/java/org/apache/phoenix/compile/WhereCompilerTest.java

```diff
@@ -1026,61 +1026,4 @@ public void testScanCaching_CustomFetchSizeOnStatement() throws SQLException {
         assertEquals(FETCH_SIZE, pstmt.getFetchSize());
         assertEquals(FETCH_SIZE, scan.getCaching());
     }
-
```

Comment:
This test is moved to WhereOptimizerTest.testLastPkColumnIsVariableLengthAndDescBug5307